src/python/pants/backend/project_info/rules/dependencies.py

```diff
@@ -52,22 +53,38 @@ class Dependencies(Goal):
     console: Console, addresses: Addresses, options: DependenciesOptions,
 ) -> Dependencies:
     address_strings: Set[str] = set()
+    third_party_requirements: Set[str] = set()
+
     if options.values.transitive:
         transitive_targets = await Get[TransitiveHydratedTargets](Addresses, addresses)
-        transitive_dependencies = transitive_targets.closure - set(transitive_targets.roots)
-        address_strings.update(
-            hydrated_target.address.spec for hydrated_target in transitive_dependencies
+        hydrated_targets = HydratedTargets(
+            transitive_targets.closure - set(transitive_targets.roots)
         )
     else:
         hydrated_targets = await Get[HydratedTargets](Addresses, addresses)
-        address_strings.update(
-            dep.spec for hydrated_target in hydrated_targets for dep in hydrated_target.dependencies
-        )
-
-    with options.line_oriented(console) as print_stdout:
-        for address in sorted(address_strings):
-            print_stdout(address)
 
+    for target in hydrated_targets:
+        address_string = target.address.spec
+        if address_string.startswith("3rdparty"):
```

Comment:
I don't think this is safe to do nor necessary. 3rdparty requirements don't _need_ to be in a folder called `3rdparty` - we only encourage that pattern. Does it work if you remove this check and only have the check for `if isinstance(target.adaptor, PythonRequirementLibraryAdaptor)`?

Reply:
It works without, though I do find it kind of odd to include targets like `3rdparty/python:www-authenticate` when `options.values.type` is `source`

I just checked though, and that's what we do in V1.

Reply:
The difference is `3rdparty/python:www-authenticate` vs. `www-authenticate==0.9.2`. The former is a target, the latter is a requirement string.